### ISSUE TITLE
Slider Minimum crashes the app

### ISSUE THREAD

**Issue by VladislavAntonyuk** [dotnet/maui#6305]
### Description

Setting the minimum value of Slider crashes the app

### Steps to Reproduce

1. Create new MAUI app.
2. Add slider to the Main page
```xml
<Slider Minimum="1" Maximum="20" Value="{Binding Value}" />
```
3. Start app.

### Version with bug

Release Candidate 1 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

19041

**Comment by jsuarezruiz** [comment]
We inherit this really old issue from Xamarin.Forms.

Setting first the Maximum value fix the issue:

`<Slider Maximum="20" Minimum="1"  Value="20" />`

The main question is, should we leave or change this behavior (using a CoerceValue with other changes?) cc @StephaneDelcroix

**Comment by VladislavAntonyuk** [comment]
Thank you @jsuarezruiz ! The workaround works

**Comment by pierre01** [comment]
Also the ValueChanged Event handler needs to be declared after. So to work the slider should be declared in the order below.

`<Slider Maximum="15" Minimum="5" Value="5" ValueChanged="Slider_ValueChanged"  />`

This need to be fixed!

**Comment by pierre01** [comment]
On Android it does not crash but it does not work correctly

**Comment by barjonas** [comment]
I can confirm that I just got burned by this. As a "my first .NET Android project", this was one of several exceptions that built up while designing/debugging the app, only to reveal themselves after a full publish and manual install. At least this one resulted in a .NET stack trace being written to the device's log and a positive hit on the GitHub search!

**Comment by hartez** [comment]
We should consider just dropping the validation on these fields; I can't think of any situation where it helps. 

Rather than throwing an exception if Min ever exceeds Max, just let them be whatever the user sets. The handlers should check for the nonsensical state when any of the relevant properties are updated and react by emitting a runtime warning and setting all of the platform control values to zero (or something nonfunctional but also non-crashing).

**Comment by jfversluis** [comment]
@pierre01 what is the problem with the event order?

**Comment by pierre01** [comment]
@jfversluis Actually in code (C#) the order does not matter.
I hooked the event before setting properties ... and also in another run, after, and they both work.
It seems to be an issue with the XAML ... Unless you have pushed your update since your last message.

**Comment by hartez** [comment]
Fixed by #8560
